exercises/transpose/src/example/java/Transpose.java

```diff
@@ -0,0 +1,41 @@
+public class Transpose {
+    public String transpose(String toTranspose) {
+
+        StringBuilder transposedForm = new StringBuilder();
+        String[] rows = toTranspose.split("\n");
+
+        int maxRowSize = 0;
+
+        for (int i = 0; i < rows.length; i++) {
+            if (rows[i].length() > maxRowSize) {
+                maxRowSize = rows[i].length();
+            }
+        }
+
+        padRows(rows);
+
+        for (int newRowOldCol = 0; newRowOldCol < maxRowSize; newRowOldCol++) {
+            for (int newColOldRow = 0; newColOldRow < rows.length; newColOldRow++) {
+                if (newRowOldCol < rows[newColOldRow].length()) {
```

Comment:
Yes. The way you're supposed to pad, you end up having rows on one or two test cases that are not the length of the longest row